README.md

```diff
@@ -125,7 +125,6 @@ WARNING: You have not specified enough hosts to satisfy the blueprint.
 ```
 $ yaas cluster create cluster1 --template=template1.json
 $ cat template2.json | yaas cluster create cluster2
-$ yaas cluster create cluster3
```

Comment:
We should look into this further because you could be right, but what it looks like their POSTing doesn't look like what I have in my head as a typical template.